k8s.io/configmap-www-golang.yaml

```diff
@@ -4,6 +4,26 @@ metadata:
   name: www-golang
 data:
   # Adding new entries here will make them appear as files in the deployment.
+  contrib.html: |
+    <html><head>
+      <meta name="go-import" content="k8s.io/cloud-provider-aws git https://github.com/kubernetes/cloud-provider-aws">
+      <meta name="go-source" content="k8s.io/cloud-provider-aws     https://github.com/kubernetes/cloud-provider-aws https://github.com/kubernetes/cloud-provider-aws/tree/master{/dir} https://github.com/kubernetes/cloud-provider-aws/blob/master{/dir}/{file}#L{line}">
+    </head></html>
+  contrib.html: |
+    <html><head>
+      <meta name="go-import" content="k8s.io/cloud-provider-azure git https://github.com/kubernetes/cloud-provider-azure">
+      <meta name="go-source" content="k8s.io/cloud-provider-azure     https://github.com/kubernetes/cloud-provider-azure https://github.com/kubernetes/cloud-provider-azure/tree/master{/dir} https://github.com/kubernetes/cloud-provider-azure/blob/master{/dir}/{file}#L{line}">
+    </head></html>
+  contrib.html: |
```

Comment:
Should it be `cloud-provider-gcp.html` instead of `contrib.html`?

Reply:
Done!